### ISSUE TITLE
Set include directories C++

### ISSUE THREAD

**Issue by chaitunya** [onivim/oni#2247]
Oni Version: 0.3.1
Operating System: Windows

I'm using Oni for C++. I want to set the include directories. I've checked the documentation, but it doesn't say anything about modifying the include directories.

Thanks in advance.

**Comment by Akin909** [comment]
@chaitunya I don't write `c++` (@TalAmuyal I think writes c++ ?) so not the best person to help here but what do you mean when you say set include directories, I mean what functionality should that result in changes to autocomplete or something relating to compiling the code?

**Comment by Escapingbug** [comment]
Same here, wondering how to add include path

**Comment by badosu** [comment]
I don't think we can expect a dedicated c++ plugin for Oni for now, this should be handled by a LSP.

cquery supports include autocompletion, see: https://github.com/cquery-project/cquery. 

Let us know if you're able to configure and use it with Oni, see: https://github.com/onivim/oni/issues/1016
